covenant-signer/signerservice/middlewares/content_length.go

```diff
@@ -0,0 +1,22 @@
+package middlewares
+
+import (
+	"net/http"
+)
+
+func ContentLengthMiddleware(maxBytes int64) func(http.Handler) http.Handler {
```

Comment:
we can have middleware for setting content type to json as well

Reply:
what do you mean ?